### ISSUE TITLE
Paste repeat on multiple lines

### ISSUE THREAD

**Issue by aith85** [john-guo/columnpaste#12]
What about adding this functionality?
https://github.com/microsoft/vscode/issues/230953

Sublime Text 4 does it by default

**Comment by john-guo** [comment]
This was an easy feature to add, and I’ve already included it in version 0.0.9. Now, when you select multiple lines, they’ll be copied one by one based on the number of lines you’ve selected.

**Comment by aith85** [comment]
Hi John,
Thank you very much for the update!
However it's not working as expected.
It seems it only works when you select multiple lines for paste but in a single selection.
If you select multiple places for pasting (ALT + click) more than the copied ones, it just paste 1 time even if you want to paste on more lines.

EG:
Copy:
```
**1
2**
```

Destination ( | is the cursor):
```
a|bc
a|bc
ab|c
abc|
abc
```

Result:
```
a**1**bc
a**2**bc
ab**1**c
abc**2**
abc
```

Hope it makes sense.
